users/views.py

```diff
@@ -224,9 +225,9 @@ def get_by_access_token(request):
 
 @api_view(["POST"])
 @permission_classes([AllowAny])
-def convert_third_party_access_token(request):
+def convert_api_key_to_token(request):
     """
-    Convert any third party auth token into Plio's internal access token
+    Convert an api_key that Plio created for a third party org, into Plio's internal access token
```

Comment:
thoda aur clear sochna hoga ismein kuch - API key to access token actually nahi kar rahe na? unique ID + API Key ko kar rahe - and API key can also be used for other things beyond auth - shayad end points ko bhi rename karna chaiye - something like: `get_external_auth_access_token` type.